crates/l1tx/src/filter/withdrawal_fulfillment.rs

```diff
@@ -110,6 +152,9 @@ mod test {
     use super::*;
     use crate::filter::types::{conv_deposit_to_fulfillment, OPERATOR_FEE};
 
+    // First 4 bytes of "strata"?  I am not sure what this is supposed to be.
+    const MAGIC: [u8; 4] = [b's', b't', b'r', b'a'];
```

Comment:
Bridge currently uses `alpenstrata`

Reply:
Maybe we should move https://github.com/alpenlabs/alpen/blob/b978d34155796136340224dd5f927c48a83d0f9c/crates/util/python-utils/src/constants.rs#L5-L6
to `strata-test-utils`?

Reply:
and use it here